cypress/integration/userCanReadASpecificArticle.feature.js

```diff
@@ -0,0 +1,62 @@
+describe("User can", () => {
+  beforeEach(() => {
+    cy.server();
+    cy.route({
+      method: "GET",
+      url: "http://localhost:3000/api/articles",
+      response: "fixture:articles_data.json",
+    });
+  });
+
+  context("successfully", () => {
+    beforeEach(() => {
+      cy.server();
+      cy.route({
+        method: "GET",
+        url: "http://localhost:3000/api/articles/1",
+        response: "fixture:single_article.json",
+      });
+      cy.visit("/");
+      cy.get("[data-cy='article-1']").click();
+    });
+    it("read a specific article", () => {
+      cy.get("[data-cy='article-display']").within(() => {
+        cy.get("[data-cy='title']").should(
+          "contain",
+          "Cats are better than dogs!"
+        );
+        cy.get("[data-cy='lead']").should(
+          "contain",
+          "Have you noticed how smelly dogs are? Well that..."
+        );
+        cy.get("[data-cy='body']").should(
+          "contain",
+          "is because they don't clean them self as good as cat does. Cats are always nice smelling and very well behaved. This is no lie, it is a simple truth-fact for sure!!"
+        );
+      });
+    });
+  });
+
+  context("not see an article if improper params are provided", () => {
+    beforeEach(() => {
+      cy.server();
+      cy.route({
+        method: "GET",
+        url: "http://localhost:3000/api/articles/abc",
```

Comment:
for readability of the test you should probably make the request with an id that you don't have but is still a number. The use case for this is example if an article got deleted the same second I pressed on it. 
Why you used `abc` in your backend is simply to trigger the error and make sure that it also catches if someone would make that request from another resource than your client

Reply:
We changed it to 123456.